ftl/cached/cached.py

```diff
@@ -78,17 +88,14 @@ def _fetch_lyr_shas(self, img_name):
                 lyr_shas.append(lyr['digest'])
             return set(lyr_shas)
 
-    def _compare_layers(self, lyr_shas_1, lyr_shas_2):
-        if len(lyr_shas_1) != len(lyr_shas_2):
-            raise RuntimeError("different amount of layers found when \
-                               reuploading same image")
-        lyr_diff_cnt = 0
-        for lyr_sha in lyr_shas_1:
-            if lyr_sha not in lyr_shas_2:
-                lyr_diff_cnt += 1
-            if lyr_diff_cnt > 1:
-                raise RuntimeError("more layers differed then app layer when \
-                    reuploading image")
+    def _compare_layers(self, lyr_shas_1, lyr_shas_2, offset):
+        lyr_diff = lyr_shas_1 - lyr_shas_2
```

Comment:
what if you passed these in the opposite order? e.g. `self._compare_layers(lyr_shas_2, lyr_shas_1, 1)`? do you not also need to do something like
```
lyr_diff_reverse = lyr_shas_2 - lyr_shas_1
```

Reply:
done